tests/unit/expire.tcl

```diff
@@ -491,7 +491,7 @@ start_server {tags {"expire"}} {
             {set foo1 bar PXAT *}
             {set foo1 bar PXAT *}
             {set foo1 bar PXAT *}
-            {set foo1 bar PXAT *}
+            {set foo1 bar pxat *}
```

Comment:
The second uses pxat, so get pxat.

so the tests can cover the changes in PXAT / pxat cases